src/transformers/pipelines/table_question_answering.py

```diff
@@ -335,7 +336,13 @@ def _sanitize_parameters(self, sequential=None, padding=None, truncation=None, *
             forward_params["sequential"] = sequential
         return preprocess_params, forward_params, {}
 
-    def preprocess(self, pipeline_input, sequential=None, padding=True, truncation="drop_rows_to_fit"):
+    def preprocess(self, pipeline_input, sequential=None, padding=True, truncation=None):
+        if truncation is None:
+            if self.type == "tapas":
+                truncation = "drop_rows_to_fit"
+            else:
+                truncation = "do_not_truncate"
```

Comment:
TAPEX also supports the "drop_rows_to_fit" truncation strategy.

Reply:
The `tokenizer` said it didn't and gave me the regular list.

Is the `tokenizer` that gets resolved a `BartTokenizer` maybe ? (I have seen `BartTokenizer` in some random error log, I discarded it, but maybe ti's a True bug.

Reply:
```python
from transformers import AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("microsoft/tapex-large-finetuned-wtq")
tokenizer.encode(table, truncation="drop_rows_to_fit")
```

```
ValueError: drop_rows_to_fit is not a valid TruncationStrategy, please select one of ['only_first', 'only_second', 'longest_first', 'do_not_truncate']
```

Reply:
Thanks for reporting, investigating.

Reply:
Update: you can continue with this, TAPEX doesn't support `drop_rows_to_fit` in a similar manner as TAPAS does.